miner/miner.go

```diff
@@ -425,8 +426,37 @@ func (m *Miner) mineOne(ctx context.Context, base *MiningBase) (*types.BlockMsg,
 		return nil, xerrors.Errorf("failed to get mining base info: %w", err)
 	}
 	if mbi == nil {
+		log.Warnf("mineOne: unexpectedly nil MiningBaseInfo for round %d, off tipset %d/%s", round, base.TipSet.Height(), base.TipSet.Key().String())
 		return nil, nil
 	}
+
+	// always write out a log from this point out
+	var winner *types.ElectionProof
+	lookBack := make(chan int64)
+
+	// figure this out in the background, instead of slowing down the main loop
+	go func() {
+		lb := int64(-1)
+		if netVer, err := m.api.StateNetworkVersion(ctx, base.TipSet.Key()); err == nil {
+			lb = int64(policy.GetWinningPoStSectorSetLookback(netVer))
+		}
+		lookBack <- lb
+	}()
```

Comment:
is this information really critical to log? This has been 900 since pretty much the beginning of time (for mainnet, at least) and will likely continue to be so.

Reply:
If we are not planning to change this anymore: I can just hardcode it to 900. It [did change in v3 fwiw, which yeah,,, pretty far away](https://github.com/filecoin-project/lotus/blob/v1.8.0/chain/actors/policy/policy.go#L180-L186)
We do need to keep emitting it though: it was *really* confusing for folks in slack.

Reply:
It's just that firing off a goroutine to fetch something that almost never changes is a bit annoying... but it's not terrible.

Reply:
Killed: https://github.com/filecoin-project/lotus/pull/6183/commits/594bf1dd46d44b2ac9ef53ad6d27a4efcec11c0c#diff-2371a1aa9e8a62882b35a2a217f7207ba984cb3258754083653ba77d3fa639ce